radix-engine/src/kernel/kernel_callback_api.rs

```diff
@@ -26,6 +26,7 @@ pub enum CreateNodeEvent<'a> {
 #[derive(Debug)]
 pub enum DropNodeEvent<'a> {
     Start(&'a NodeId),
+    StoreAccess(&'a StoreAccess),
```

Comment:
We should rename `StoreAccess` to `IOAccess` now since it's all I/O now :)

Reply:
Yes!